mochify/lib/resolve-spec.js

```diff
@@ -6,6 +6,10 @@ const glob = promisify(require('glob'));
 exports.resolveSpec = resolveSpec;
 
 async function resolveSpec(spec = 'test/**/*.js') {
+  if (typeof spec === 'object' && typeof spec.pipe === 'function') {
```

Comment:
I found it nicer to handle such situations at `lib` level instead of `index.js` as index.js already contains a lot of control flow and it would have gotten even more.

Reply:
Yeah, I also get the feeling it's growing quite a bit.

Maybe it's time to introduce more functions like `run` that handle parts of the control flow. E.g. `configure`, `launch`, `run` and `shutdown`. Just a random morning thought though 🙃